_includes/header.html

```diff
@@ -18,8 +18,8 @@
           <img
             class="syn-ml-8"
             src="/v1/config/logo"
+            alt="Algorithmia Logo"
             style="max-height: 24px; max-width: 100%"
-            onerror="this.onerror=null; this.src=`{{ '/images/algo-purple.svg' | relative_url }}`;"
```

Comment:
How come you removed this?

Reply:
When I ran a local server with the CSP enabled, it threw a fit because of the inline script. In order to fix this, I'd have to create a new JS file and load that. It didn't seem worth it to go through the trouble since the file won't 404 in prod and is really just being used for local dev.

Reply:
Could we just skip this when the CSP is enabled? It'd be nice to have it function locally without the CSP still.

Reply:
I take it the issue is that, if the docs app is running locally, there isn't a proxy to ferry the `/v1/config/logo` request back to test, and so the image errors? Rather than bifurcate on the CSP, could we just toggle on whether dev mode is active, if `true`, hardcode `https://test.algorithmia.com/v1/config/logo`?

Reply:
Yeah, that could work

Reply:
Good call. Updated